app/models/track.rb

```diff
@@ -41,4 +41,9 @@ def about
   def repo
     Git::ExercismRepo.new(repo_url)
   end
+
+  def accepting_new_students?
+    median_wait_time &&
+    median_wait_time < 604800 # 1 week
```

Comment:
Should we rename to `accepting_new_mentees?` for clarity?

Reply:
We've started calling them students throughout the site now, so I think I'm happy with that. But good shout.